src/services/s3/backend.rs

```diff
@@ -347,6 +351,7 @@ impl Accessor for Backend {
             .await
             .map_err(|e| parse_get_object_error(e, "read", &args.path))?;
 
+        self.metrics().incr_read(resp.content_length() as u64);
```

Comment:
It's incorrect as we could drop the request during reading, maybe we can calculate it inside `S3ByteStream` which does the real IO.